### PR TITLE
Add Everything

### PR DESCRIPTION
**Pull request by carstenr**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+<div id="app">
+    <div class="page-container">
+        <md-app md-mode="overlap">
+            <md-app-toolbar class="md-primary md-large">
+                <div class="md-toolbar-row">
+                    <md-button class="md-icon-button" @click="menuVisible = !menuVisible">
```
Comment (chripu) [inline index.html]: Menu is not needed here.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+        computed: {
+            hasError: function() {
+                if (this.input === '') {
+                    return false;
+                }
+                let i = parseInt(this.input);
+                if (typeof i !== 'number' || !Number.isInteger(i)) {
+                    return true;
+                }
+                if (this.type === 'K' && i < 0) {
```
Comment (chripu) [inline index.html]: I would prefer additonal parentheses in this cases:
`if ((this.type === 'K') && (i < 0)) {`

**Comment by chripu** [comment]
Good job.  I only noted some personal perferences.

File: index.html (hunk 1 above)
Comment (carstenr) [reply index.html]: Indeed

**Pull request closed without merging**